.github/workflows/bump-elastic-stack-version.yml

```
@@ -35,28 +35,10 @@ jobs:
           echo "UPDATECLI_ACTION=apply" >> $GITHUB_ENV
 
       - name: Install Updatecli in the runner
-        uses: updatecli/updatecli-action@v2.62.0
+        uses: updatecli/updatecli-action@3a8785d88ec4fa03d86521a181f37c0e74627463 #v2.64.0
```

I prefer the use of tags, they are easier to find in the releases. Is there any advantage or reason to use the commit hash instead?
```suggestion
        uses: updatecli/updatecli-action@v2.64.0
```

Or dependabot also adds the comment when updating the action?

---

>  Is there any advantage or reason to use the commit hash instead?

Tags are mutable and full sha commits are considered the way to do it as explained [here](https://docs.github.com/en/enterprise-cloud@latest/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions)

>> Pin actions to a full length commit SHA
Pinning an action to a full length commit SHA is currently the only way to use an action as an immutable release. Pinning to a particular SHA helps mitigate the risk of a bad actor adding a backdoor to the action's repository, as they would need to generate a SHA-1 collision for a valid Git object payload. When selecting a SHA, you should verify it is from the action's repository and not a repository fork.

> Or dependabot also adds the comment when updating the action?

Dependabot does it , see https://github.com/elastic/oblt-actions/pull/105/files

---

Perfect, thanks!